x-pack/plugins/observability_solution/dataset_quality/public/components/flyout/flyout.tsx

```diff
@@ -58,7 +68,7 @@ export default function Flyout({ dataset, closeFlyout }: FlyoutProps) {
         <EuiSkeletonRectangle width="100%" height={80} />
       ) : (
         <>
-          <Header dataStreamStat={dataset} />
+          <Header titleAndLinkDetails={titleAndLinkDetails} />
```

Comment:
should we have a loading state for the header as for example we do for APM services (when trying to get agent name icons)? In this case we will wait until we get the integration

Reply:
perfect 👍🏼